erpnext/hr/doctype/employee_checkin/employee_checkin.py

```diff
@@ -104,6 +104,13 @@ def mark_attendance_and_link_log(logs, attendance_status, attendance_date, worki
 				'in_time': in_time,
 				'out_time': out_time
 			}
+
+			hd_status, p_status, a_status = get_applicable_status()
```

Comment:
```suggestion
			half_day_status, present_status, absent_status = get_applicable_status()
```
Can you change the variable names for better context? Lookout for these variables in the whole file.